### PR TITLE
Portfolio main section

### PR DESCRIPTION
**Pull request by lengushuru**
##description

    -The technology categories ("Languages", "Frameworks" and "Skills") are wrapped in one ul element and each one of them is in a li.

   -works are now displayed using display: grid and each work is displayed with display: flex.

  -all IMG tags in work are replaced with background image

all suggestion from previous reviewers has been implemented

### CODE REVIEW

**Review by NduatiKagiri — CHANGES_REQUESTED** [review]
Hi @lengushuru , 

Good job so far 💯 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights ✔️
✔️ Descriptive documentation
✔️ Project requirements met
✔️ Linters are passing

## Required Changes ♻️

- Please ensure you apply the background images as background images and not by using `img` tag.
- Kindly use `button`  or  `a` tags for "See Project" since this should be links to do action and not just texts.
- The "Let's Connect" heading should use a `hx` tag e.g. `h2` since its a heading and not a paragraph
- The technology categories ("Languages", "Frameworks" and "Skills") should all be wrapped in one `ul` element and each one of them in a `li`, but your "Languages" category is not in the `ul` element. Kindly fix that
- This project requires your works to be displayed using `display: grid` and each work should be displayed with `display: flex`, but you have used grid for both of them. Kindly use flex box for each of the work's contents

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-  In your future projects, please always ensure your changes appear on the PR for easier reviews

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lengushuru** [comment]
@NduatiKagiri thank you for your feedback. I don't understand well point number one,  of applying background images I used only one background image the rest are not backgrounded images. please clarify this. again I have used a CSS grid for every work section together with CSS flex.

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @lengushuru 😄  ,

Good job so far 💯
There are some issues that you still need to work on to go to the next project but you are almost there!

Highlights ✔️
✔️ Descriptive documentation
✔️ Linters are passing

## Required Changes ♻️

- [x] Kindly go through this [brief turtorial](https://www.w3schools.com/cssref/pr_background-image.php) for an idea on how to apply background image in CSS. 😄 

- [x] Please ensure you do this as the previous reviewer requested. 
 > The technology categories ("Languages", "Frameworks" and "Skills") should all be wrapped in one ul element and each one of them in a li, but your "Languages" category is not in the ul element. Kindly fix that

- [x] This project requires your works to be displayed using display: grid and each work should be displayed with display: flex, but you have used grid for both of them. Kindly use flex box for each of the work's contents.


## Optional suggestion

[optional] Please take note for subsequent reviews, ensure that your changes are made in the feature branch not the default branch. As it is, I have limited interaction with your PR.

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:

In your future projects, please always ensure your changes appear on the PR for easier reviews
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 1]
```diff
@@ -135,13 +135,13 @@ <h1>Tonic</h1>
...
-                     <a href="#"><p>See Project</p></a>>
+                     <a href="#"><p>See Project</p></a>
                  </div>
             </div>  
         </div>
         <div class="portfolio">
-            <div class="snapshot">
-                <img src="./images/portfolio3.png" alt="fourth portfolio">
+            <div class="snapshot4">
+                <!-- <img src="./images/portfolio3.png" alt="fourth portfolio"> -->
```
Comment (Debas-31) [inline index.html:144]: [OPTIONAL] - Please clean all commented code in your HTML file to be your code clean and professional.

**Review by Debas-31 — APPROVED** [review]
> ## Hi @lengushuru  :wave: , 

# Status: Approved :green_circle:  

Great work on making the changes requested by a previous reviewer. Your project is complete !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Linters setup :heavy_check_mark:
- GitHub flow :heavy_check_mark: 
- Project working :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**